### DISCUSSION TITLE
Cannot use enum for queryString

### DISCUSSION THREAD

**Discussion by mschop** [livewire/livewire#5230] [7. Bugs (archived)]
Hi,

using a backed enum in a query string does not work.

The enum is not serialized ideally in the query string. This is contained in the query string:
`module[name]=DASHBOARD&module[value]=dashboard` 

And that does not work when reloading the page:

```
Cannot assign array to property App\Http\Livewire\Admin\App::$module of type App\Http\Livewire\Admin\Models\Module
```

Expected behavior:

Only the enum value (backed value) should be serialized in the query string (so just `module=dashboard`).

Additionally enums should not break on page refresh.

Best Regards
mschop

**Comment by mjawad096** [comment]
@mschop facing same issue, I've submitted a PR for this #5705
